consensus/src/consensus_observer/observer/subscription.rs

```diff
@@ -270,18 +314,38 @@ pub fn sort_peers_by_distance_and_latency(
         );
     }
 
+    // Log the sorted peers
+    info!(
```

Comment:
how frequent is this log? does it need to be sampled?

Reply:
This will only get called every few minutes (at most), so we should be good not to sample it 😄